addons/website_sale/views/templates.xml

```diff
@@ -4357,8 +4357,8 @@
         >
             <ol
                 t-att-class="'carousel-indicators '
-                    + (indicators_list_class or '')
-                    + ('justify-content-center ' if website.product_page_image_width == '100_pc' else '')
+                    + ((indicators_list_class + ' ') if indicators_list_class else '')
+                    + ('justify-content-center' if website.product_page_image_width == '100_pc' else '')
```

Comment:
I know we are exceeding the char guidelines but having to concatenate an empty string to a list of classes to avoid potential side effect with the next condition makes me feel it would be way easier to use a `t-attf-class`

```suggestion
                t-attf-class="carousel-indicators position-static pt-2 pt-lg-0 mx-auto my-0 {{indicators_list_class}} {{website.product_page_image_width == '100_pc' and 'justify-content-center'}}"
```